### PR TITLE
Docs API Delete An Order

### PR DESCRIPTION
**Pull request by at-vudang95**

### CODE REVIEW

File: docs/api-references/order.md [hunk 1]
```diff
@@ -156,3 +156,45 @@ Update order from client.
 	"success": true
 }
 ```
+### `DELETE` Delete An Order
+```
+/api/orders/{id}/getItems
```
Comment (ghost) [inline docs/api-references/order.md]: I don't understand `getItems` `URI`

File: docs/api-references/order.md [hunk 2]
```diff
@@ -156,3 +156,45 @@ Update order from client.
...
+{
+  "data": {
+    "id": 74,
+    "user_id": 3,
+    "created_at": "1999-02-16 03:27:10",
+    "deleted_at": "1999-02-16 03:27:10",
+    "total_price": 92500,
+    "order_items": [
+    	{
+    		"id": 23,
```
Comment (ghost) [inline docs/api-references/order.md]: Please use clear indent level.

**Review by ghost — COMMENTED** [review]
Please check my comment

File: docs/api-references/order.md (hunk 1 above)
Comment (at-vudang95) [reply docs/api-references/order.md]: I mistake, I am sorry.

**Comment by ghost** [comment]
LGTM 🔍

**Pull request merged**